### PR TITLE
move pcrelate

### PR DESCRIPTION
**Pull request by catoverdrive**

### CODE REVIEW

File: python/hail/docs/methods/index.rst [hunk 1]
```diff
@@ -15,6 +15,7 @@ Methods
     hail.methods.grm
     hail.methods.pca
     hail.methods.hwe_normalized_pca
+    hail.methods.pca_relate
```
Comment (konradjk) [inline python/hail/docs/methods/index.rst]: ➡️ `pc_relate`

Comment (catoverdrive) [reply python/hail/docs/methods/index.rst]: Womp

File: python/hail/methods/statgen.py [hunk 2]
```diff
@@ -354,6 +354,270 @@ def pca(entry_expr, k=10, compute_loadings=False, as_array=False):
...
+
+    Notes
+    -----
+    The PC-Relate method is described in "Model-free Estimation of Recent
+    Genetic Relatedness". Conomos MP, Reiner AP, Weir BS, Thornton TA. in
+    American Journal of Human Genetics. 2016 Jan 7. The reference
+    implementation is available in the `GENESIS Bioconductor package
+    <https://bioconductor.org/packages/release/bioc/html/GENESIS.html>`_ .
+
+    :meth:`methods.pc_relate` differs from the reference
```
Comment (danking) [inline python/hail/methods/statgen.py]: this should be `:func:`

**Comment by danking** [comment]
@catoverdrive needs rebase

**Pull request merged**